src/test/java/org/xstefank/YamlConfigTest.java

```diff
@@ -16,7 +17,9 @@ public void testValidTemplateConfig() {
     @Test
     public void readTokenTest() {
         String pathToTestConfig = YamlConfigTest.class.getClassLoader().getResource("testConfig.properties").getPath();
+        PropertiesLoader propertiesLoader = PropertiesLoader.getInstance();
+        propertiesLoader.savePropertiesFromFile(pathToTestConfig);
```

Comment:
`PropertiesLoader` is obviously defined by the loaded properties file, such dependencies are typically passed by the constructor. Please explain why the singleton is needed here